Intersect.Server/Entities/Player.cs

```diff
@@ -3605,7 +3605,8 @@ public void CraftItem(Guid id)
                     invbackup.Add(item.Clone());
                 }
 
-                var mCraftItem = CraftBase.Get(id);
+                var craftItem = CraftBase.Get(id);
+                var finalItem = ItemBase.Get(craftItem.ItemId);
```

Comment:
This is on me for having a brain fart earlier but `craftItem` should be `craft`, `finalItem` should be `item`.

Reply:
haha lol, okay